django-s3-file-field-client/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: "./src/client.ts",
+  output: {
+    path: path.resolve("build"),
+    filename: "joiist.js",
+    libraryExport: "Joiist",
+    libraryTarget: "umd"
+  },
+  resolve: {
+    // Add `.ts` and `.tsx` as a resolvable extension.
+    extensions: [".ts", ".tsx", ".js"]
+  },
+  module: {
+    rules: [
+      // all files with a `.ts` or `.tsx` extension will be handled by `ts-loader`
+      {
+        test: /\.tsx?$/,
+        loader: "ts-loader",
+        options: {
+          configFile: "tsconfig.lib.json"
+        }
+      },
+      {
+        test: /\.s[ac]ss$/i,
```

Comment:
I think you can drop this. The library should not touch the DOM at all, it should never have CSS.